test/ruby_lsp_rails/runner_client_test.rb

```diff
@@ -4,6 +4,9 @@
 require "test_helper"
 require "ruby_lsp/ruby_lsp_rails/runner_client"
 
+# tests are hanging in CI. https://github.com/Shopify/ruby-lsp-rails/issues/348
```

Comment:
Do we still need to skip this test on Windows? If so, let's ship this, but please prioritize unskipping it so that we don't go long with skipped tests.